packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -639,6 +703,79 @@ const processTemplate = (relFile, inputDir, outputDir, context) => {
     }
 }
 
+/**
+ * Process the Application Extensions into the application-extensions directory.
+ *
+ * @param appExtensions - An array of the Application Extension names.
+ * @param extractAppExtensions - A boolean indicating whether to extract the Application Extensions code from the npm package
+ * @param appExtensionsDir - The path to the application-extensions directory.
+ */
+const processAppExtensions = (
+    appExtensions = [],
+    extractAppExtensions = false,
+    appExtensionsDir
+) => {
+    if (appExtensions.length > 0 && extractAppExtensions) {
+        appExtensions.forEach((appExtensionName) => {
+            const appExtensionTmp = fs.mkdtempSync(
+                p.resolve(os.tmpdir(), `extract-${appExtensionName.replace('@salesforce/', '')}`)
```

Comment:
I guess we are doing this kind of replacement because either we didn't recursively crest a folder for the content or something about the @ symbol in a folder name?

Let's create a follow up pr to address this, I don't want to see these constant at salesforce strings because it's not gonna work with future none "at" namespaces